client/main.lua

```diff
@@ -308,37 +317,103 @@ local function deliver()
         exports.scully_emotemenu:cancelEmote()
         ClearPedTasks(cache.ped)
         hasBox = false
-        currentCount += 1
-        if currentCount == currentLocation.dropcount then
-            delivering = false
-            showMarker = false
+        currentLocation.currentCount += 1
+        lib.print.debug('count: '..currentLocation.currentCount..'/'..currentLocation.dropCount)
```

Comment:
If you use commas to separate the terms, it will apply a tab and encode them so that you can pretty print things like tables, or nil values.
```suggestion
        lib.print.debug('count:', currentLocation.currentCount, '/', currentLocation.dropCount)
```